index.php

```diff
@@ -398,15 +404,24 @@ function day_report($location = false)
 $lastuid = get_parameter('lastuid');
 $password = get_parameter('password');
 
-if (!preg_match('/^[a-z_0-9]{0,8}$/', $uid)) {
```

Comment:
This check was added for security reasons. Simply removing it without a replacement might be a bad idea. Of course the check should be part of the local code as the rules differ for different installations.

Reply:
Yes, that is why I had already added it to our local configuration. I'll leave it in as an example then.